dev/archery/archery/crossbow/core.py

```diff
@@ -546,8 +570,10 @@ def github_overwrite_release_assets(self, tag_name, target_commitish,
                         'Unsupported upload method {}'.format(method)
                     )
 
-    def github_pr(self, title, head=None, base="master", body=None,
+    def github_pr(self, title, head=None, base=None, body=None,
                   github_token=None, create=False):
+        # Default value for base is the default_branch name()
+        base = self.default_branch_name() if base is None else base
```

Comment:
Do we need `()` here?

Reply:
Thanks for catching this! You're right, since it's a computed property, it does not need parens. I've removed them!